packages/x-charts-pro/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { LicenseInfo as LicenseInfoExport } from '@mui/x-license';
+/**
+ * @deprecated Use `@mui/x-license` package instead:
+ * @example import { LicenseInfo } from '@mui/x-license';
+ */
+export class LicenseInfo extends LicenseInfoExport {}
```

Comment:
Do we really want to add this if its deprecated?
We don't have it on Tree View, we should of course be consistent across the 2 packages whatever we end up deciding.

Reply:
I never touched to those license files. I just copy pasted those lines from the date picker pro

Reply:
In v8 we will remove those deprecated exports from pickers and grid
And everybody will just import from the license package